### PR TITLE
Add accessibility changes to 

### LINKED ISSUE #36: Pass Chrome Audit tests
- pass with green the Accessibility tests of the Chrome Audits
- pass with green the Best Practices tests of the Chrome Audits
- pass with green the SEO tests of the Chrome Audits

### PR DESCRIPTION
**Pull request by christopher77**
**this feature contains:**

- Add attributes into labels in html pages (index , login )to pass the SEO tests
- Add attributes into labels in html pages (index , login )to pass the Accessibility tests
- Change line code in functions in main.js file and login.css file to pass the Best Practices tests

closed #36

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ function initSocket() {
 
   socket.addEventListener('message', event => {
     const newMessage = JSON.parse(event.data);
-    messages.push(newMessage);
+    messages.push(newMessage); ///ask permi o en el header
```
Comment (condef5) [inline public/js/main.js]: You could explain we more about this please?

File: public/js/main.js [hunk 2]
```diff
@@ -74,11 +74,11 @@ function test() {
 }
 
 function renderUsername() {
-  $username.innerText = currentUser.name;
+  $username.innerText = currentUser ? currentUser.name : '';
```
Comment (byteconde) [inline public/js/main.js]: Use of ternaries are an excellent option to reorder the code

Comment (byteconde) [reply public/js/main.js]: Good job @christopher77

File: public/js/main.js (hunk 1 above)
Comment (byteconde) [reply public/js/main.js]: Oh, little mistake. Could you please delete that duplicated line?

Comment (christopher77) [reply public/js/main.js]: I forgot it, that is a guidelines to future improves =)

**Comment by condef5** [comment]
Good job, all tests are green :)

File: public/js/main.js [hunk 3]
```diff
@@ -74,11 +74,11 @@ function test() {
...
 }
 
 function renderChannel() {
-  const channels = JSON.parse(localStorage.getItem('channels'));
+  const channels = JSON.parse(localStorage.getItem('channels')) || [];
```
Comment (test0n3) [inline public/js/main.js]: In this part... shouldn't `general` channel be by default? Or is it added somewhere else?

File: public/login.html [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
-
-<body>
-  <form id="loginForm">
-    <input type="text" required placeholder="Enter username 👀" name="username" id="username" />
-    <input type="submit" value="Room In" />
-  </form>
-  <script src="js/login.js"></script>
-</body>
-
+  <body>
```
Comment (test0n3) [inline public/login.html]: Missing `<h1>` tag. Add it please for semantic and SEO purposes.

File: public/js/main.js (hunk 3 above)
Comment (christopher77) [reply public/js/main.js]: oh shit

File: public/login.html (hunk 4 above)
Comment (christopher77) [reply public/login.html]: yes, I'll do it

**Pull request merged**